test/minikube/minikube_domain_resync_test.go

```diff
@@ -311,6 +377,43 @@ func TestDomainResync(t *testing.T) {
 	}, 300*time.Second)
 }
 
+func TestLoadBalancerConfigurationResync(t *testing.T) {
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"admin.lbConfig.prefilter":              "not(label(region tiebreaker))",
+			"admin.lbConfig.default":                "random(first(label(node node1) any))",
+			"admin.lbConfig.policies.zone1":         "round_robin(first(label(zone zone1) any))",
+			"admin.lbConfig.policies.nearest":       "random(first(label(pod ${pod:-}) label(node ${node:-}) label(zone ${zone:-}) any))",
+			"admin.lbConfig.fullSync":               "true",
+			"database.sm.resources.requests.cpu":    "0.25",
+			"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.te.resources.requests.cpu":    "0.25",
+			"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.lbConfig.prefilter":           "not(label(zone DR))",
+			"database.lbConfig.default":             "random(first(label(node ${NODE_NAME:-}) any))",
+			"nuodb.image.tag":                       "4.2-mod",
```

Comment:
what is this tag? This is probably left over from your manual testing.

Reply:
Thanks! I had problems with this when running the test on the internal CI.